api/proto/teleport/assist/v1/assist.proto

```diff
@@ -150,6 +151,22 @@ message GetAssistantEmbeddingsResponse {
   repeated EmbeddedDocument embeddings = 1;
 }
 
+// SearchUnifiedResourcesRequest is a request to search for one or more resource kinds using similiarity search.
+message SearchUnifiedResourcesRequest {
+  // query is the query used for similarity search.
+  string query = 2;
```

Comment:
nit: What happened with 1?

Reply:
old field i had during dev that had nr 1. fixed nw.